dc_utils/__init__.py

```diff
@@ -1 +1 @@
-__version__ = "0.0.4"
+__version__ = "0.0.5"
```

Comment:
One last comment - this is fine here for now, but a better workflow for versions it to do the commit directly on master right before publishing a new release.

Imagine we had 5 open PRs that could get merged at different times. We might not bump the version until 3 of them are in, but if all of them changed the version then it would get confusing! And if the other 2 changed the version to the newer published version number then it would be even more confusing!

I see version bumping as being coupled with releasing, not PRs, if you see what I mean. (but in this case, we only have 1 PR so it's not a problem!)

Reply:
Ah yes, I remember you saying this now. Makes complete sense, thanks.

Reply:
This is another thing for the handbook!